.gitmodules

```diff
@@ -16,8 +16,8 @@
 	branch = main
 [submodule "src/drivers/gps/devices"]
 	path = src/drivers/gps/devices
-	url = https://github.com/PX4/PX4-GPSDrivers.git
-	branch = main
+	url = https://github.com/slgrobotics/PX4-GPSDrivers.git
+	branch = RTK-HPPOSLLH
```

Comment:
Can you rebase, squash the commits, update the gps submodule and revert this? Then it should be good to merge.